docs/wip/traffic/permissive_traffic_policy_mode.md

```diff
@@ -0,0 +1,343 @@
+---
+title: "Permissive traffic policy mode"
+description: "Permissive traffic policy mode"
+type: docs
+aliases: ["permissive_traffic_policy_mode.md"]
+---
+
+# Permissive traffic policy mode
+Permissive traffic policy mode in OSM is a mode where [SMI][1] traffic policy enforcement is bypassed. In this mode, OSM automatically discovers services that are a part of the service mesh and programs traffic policy rules on each Envoy proxy sidecar to be able to communicate with these services.
+
+## When to use permissive traffic policy mode
+Since permissive traffic policy mode bypasses [SMI][1] traffic policy enforcement, it is suitable for use when connectivity between applications within the service mesh should flow as before the applications were enrolled into the mesh. This mode is suitable in environments
+where explicitly defining traffic policies for connectivity between applications it not feasible.
+
+A common use case to enable permissive traffic policy mode is to support gradual onboarding of applications into the mesh without breaking application connectivity. Traffic routing between application services is automatically set up by OSM controller through service discovery. Wildcard traffic policies are set up on each Envoy proxy sidecar to allow traffic flow to services within the mesh.
+
+The alternative to permissive traffic policy mode is SMI traffic policy mode, where traffic between applications is denied by default and explicit SMI traffic policies are necessary to allow application connectivity. When policy enforcement is neccesary, SMI traffic policy mode must be used instead.
```

Comment:
```suggestion
The alternative to permissive traffic policy mode is SMI traffic policy mode, where traffic between applications is denied by default and explicit SMI traffic policies are necessary to allow application connectivity. **When policy enforcement is necessary, SMI traffic policy mode must be used**.
```

Reply:
This emphasis will go into the `_index.md` file. Don't think its necessary here.